src/presentation/home/homePage.js

```diff
@@ -1,16 +1,75 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import '../../App.css'
+import RaisedButton from 'material-ui/RaisedButton'
+import AppBar from 'material-ui/AppBar'
+import Avatar from 'material-ui/Avatar'
+import {Card, CardActions, CardHeader, CardMedia, CardTitle, CardText} from 'material-ui/Card'
+import Checkbox from 'material-ui/Checkbox'
+import TextField from 'material-ui/TextField'
+import Toggle from 'material-ui/Toggle'
+import {Toolbar, ToolbarGroup, ToolbarSeparator, ToolbarTitle} from 'material-ui/Toolbar'
+import {List, ListItem} from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+import Subheader from 'material-ui/Subheader';
+
+
+
+// Material UI SVG icons
+import AccountIcon from 'material-ui/svg-icons/action/account-circle'
+import HomeIcon from 'material-ui/svg-icons/action/home';
+import ListIcon from 'material-ui/svg-icons/action/list';
+import PeopleIcon from 'material-ui/svg-icons/social/people';
+import PinIcon from 'material-ui/svg-icons/maps/pin-drop';
+import SearchIcon from 'material-ui/svg-icons/action/search'
+import StoreIcon from 'material-ui/svg-icons/action/store'
+
+// Image importing would only work via require
+const Logo = require('./film-indy-logo.jpg')
 
 const homePage = props => (
-  <div >
-    <p className="App-intro">
-            To get started, edit <code>src/App.js</code> and save to reload.
-    </p>
-    <button onClick={props.onButtonClick}>CLICK ME </button>
-    <h1>
-      {`I've been clicked ${props.timesButtonPressed} times`}
-    </h1>
+  <div>
+    <AppBar
```

Comment:
This should live in `App.js` so it can be the same across every page.  We don't want to copy and paste this code into every page we have.